samples/test/features/totp-signup.feature

```diff
@@ -10,7 +10,7 @@ Feature: TOTP Support (Google Authenticator) Sign Up
     When she fills out her First Name
       And she fills out her Last Name
       And she fills out her Email
-      And she fills out her Age
+      # And she fills out her Age
```

Comment:
👍  I feel we should remove it from this from the spec, custom attribute is irrelevant here.